tutorials/intermediate/imshow_extent.py

```diff
@@ -263,3 +257,6 @@ def generate_imshow_demo_grid(extents, xlim=None, ylim=None):
 
 generate_imshow_demo_grid(extents=[None] + extents,
                           xlim=(-2, 8), ylim=(-1, 6))
+
```

Comment:
This should be zero or at most one empty line. The two statements are rendered as a standalone code block, and two empty lines is clearly too much:

~~~
generate_imshow_demo_grid(extents=[None] + extents,
                          xlim=(-2, 8), ylim=(-1, 6))


plt.show()
~~~

Reply:
sure